samples/vision/face_samples.py

```diff
@@ -1,80 +1,694 @@
-import os.path
-
 from azure.cognitiveservices.vision.face import FaceClient
-from azure.cognitiveservices.vision.face.models import FaceAttributeType, Gender
 from msrest.authentication import CognitiveServicesCredentials
+from azure.cognitiveservices.vision.face.models import FaceAttributeType, HairColorType, TrainingStatusType, Person
+import os
+import uuid
+import time
 
 SUBSCRIPTION_KEY_ENV_NAME = "FACE_SUBSCRIPTION_KEY"
 FACE_LOCATION = os.environ.get("FACE_LOCATION", "westcentralus")
 
 IMAGES_FOLDER = os.path.join(os.path.dirname(os.path.realpath(__file__)), "images", "Face")
 
-def face_detect(subscription_key):
-    """ImageAnalysisInStream.
+def detect_faces(face_client, image_url):
+    """DetectFaces.
+
+    This will detect the faces found in the image with url image_url using the provided FaceClient instance and return a list of faces detected in the image.
+    """
+
+    detected_faces = face_client.face.detect_with_url(image_url, True, False, None, None, False)
```

Comment:
Never do positional arguments with booleans and None, use keyword arguments only:
Not:
```python
foo(True)
```
but
```python
foo(bar=True)
```